assets/scss/components/stencil/navUser/_navUser.scss

```diff
@@ -123,6 +123,27 @@
     }
 }
 
+span.navUser-item-social {
+    display: none;
+
+    @include breakpoint("medium") {
+        display: block;
+        margin-top: 10px;
+        padding-right: 10px;
+    }
+}
+
+span.navUser-item-social-break {
+    display: none;
```

Comment:
Can you write this statement without targeting a `span` specifically? We try not to style by tag, this is a bit too specific.